samples/Ark.ReferenceProject/Core/Ark.Reference.Core.Common/JsonContext/CoreApiJsonSerializerContext.cs

```diff
@@ -0,0 +1,35 @@
+using Ark.Reference.Core.Common.Dto;
+using Ark.Reference.Core.Common.Enum;
+using Ark.Tools.Core;
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace Ark.Reference.Core.Common.JsonContext;
+
+/// <summary>
+/// JSON serialization source generation context for Core API
+/// </summary>
+[JsonSourceGenerationOptions(
+    WriteIndented = false,
+    PropertyNamingPolicy = JsonKnownNamingPolicy.CamelCase,
+    DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull,
+    GenerationMode = JsonSourceGenerationMode.Default,
+    UseStringEnumConverter = true)]
+[JsonSerializable(typeof(Ping.V1.Create), TypeInfoPropertyName = "PingV1Create")]
+[JsonSerializable(typeof(Ping.V1.Update), TypeInfoPropertyName = "PingV1Update")]
+[JsonSerializable(typeof(Ping.V1.Output), TypeInfoPropertyName = "PingV1Output")]
+[JsonSerializable(typeof(PagedResult<Ping.V1.Output>), TypeInfoPropertyName = "PagedResultPingV1Output")]
+[JsonSerializable(typeof(IEnumerable<Ping.V1.Output>), TypeInfoPropertyName = "IEnumerablePingV1Output")]
```

Comment:
IEnumerable should not be needed. neither string/bool.

Only root-level types are required. Take the Serializable types from the Handler definitions, both CQRS handlers and message handlers.